lib/WebpackOptionsApply.js

```diff
@@ -588,7 +588,6 @@ class WebpackOptionsApply extends OptionsApply {
 					break;
 				}
 				default:
-					// @ts-ignore never is expected here
 					throw new Error(`Unknown cache type ${options.cache.type}`);
```

Comment:
```suggestion
					throw new Error(`Unknown cache type ${cacheOptions.type}`);
```

So you can use expect-error